slack_sdk/web/client.py

```diff
@@ -1028,295 +1588,497 @@ def calls_participants_remove(
         **kwargs,
     ) -> SlackResponse:
         """Registers participants removed from a Call.
-
-        Args:
-            id (str): id returned when registering the call using the calls.add method.
-            users: (list): The list of users to remove as participants in the Call.
+        https://api.slack.com/methods/calls.participants.remove
         """
         kwargs.update({"id": id})
         _update_call_participants(kwargs, users)
         return self.api_call(
             "calls.participants.remove", http_verb="POST", params=kwargs
         )
 
-    def calls_update(self, *, id: str, **kwargs) -> SlackResponse:  # skipcq: PYL-W0622
+    def calls_update(
+        self,
+        *,
+        id: str,
+        desktop_app_join_url: Optional[str] = None,
+        join_url: Optional[str] = None,
+        title: Optional[str] = None,
+        **kwargs,
+    ) -> SlackResponse:  # skipcq: PYL-W0622
         """Updates information about a Call.
-
-        Args:
-            id (str): id returned by the calls.add method.
+        https://api.slack.com/methods/calls.update
         """
-        kwargs.update({"id": id})
+        kwargs.update(
+            {
+                "id": id,
+                "desktop_app_join_url": desktop_app_join_url,
+                "join_url": join_url,
+                "title": title,
+            }
+        )
         return self.api_call("calls.update", http_verb="POST", params=kwargs)
 
-    def channels_archive(self, *, channel: str, **kwargs) -> SlackResponse:
-        """Archives a channel.
+    # --------------------------
+    # Deprecated: channels.*
+    # --------------------------
 
-        Args:
-            channel (str): The channel id. e.g. 'C1234567890'
-        """
+    def channels_archive(
+        self,
+        *,
+        channel: str,
+        **kwargs,
+    ) -> SlackResponse:
+        """Archives a channel."""
         kwargs.update({"channel": channel})
+        kwargs = _filter_none_values(kwargs)
```

Comment:
For `json=kwargs` API calls, having `None` value in dict can result in `null` value in JSON data. The Slack API server-side often does not work with the data. Thus, we remove `None` values here. For `params` / `data`, we don't need to worry about it.